website/content/_index.md

```diff
@@ -50,6 +50,7 @@ MetalLB requires the following to function:
   capable of
   speaking
   [BGP](https://en.wikipedia.org/wiki/Border_Gateway_Protocol).
+- If using the layer 2 operating mode, the host firewall must allow port 7946 (TCP & UDP) for gossip discovery communication between nodes, as required by [memberlist](https://github.com/hashicorp/memberlist).
```

Comment:
Even if memberlist is only used for layer 2, it’s enabled regardless of the configured mode.

I think you should remove this part: `If using the layer 2 operating mode, `.

Reply:
You're right, at the moment memberlist is initialized regardless of the protocol in use. We plan to change that (see https://github.com/metallb/metallb/issues/614).

I'll remove the layer 2 mention for now and will add a TODO item to re-add it once we move the memberlist initialization into the layer 2 part of the code.